docker/kali109-x64/Dockerfile

```diff
@@ -3,7 +3,8 @@
 FROM rapid7/build:kali109_64
 MAINTAINER Rapid7 Release Engineering <r7_re@rapid7.com>
 
-RUN echo "deb http://old.kali.org/kali moto main non-free contrib" > /etc/apt/sources.list
+# package/ruby: needs gcc and host gcc >= 4.9, available only in Sana - https://www.kali.org/blog/kali-moto-eol/
```

Comment:
Fixing the compile error:

> thread_pthread.h:108:43: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'struct'
     RUBY_EXTERN RB_THREAD_LOCAL_SPECIFIER struct rb_execution_context_struct *ruby_current_ec;

https://lists.buildroot.org/pipermail/buildroot/2021-October/626816.html